docs/dev/news/3641.feat

```diff
@@ -0,0 +1 @@
+Expose non-relaxed definite assignment (`/definiteAssignment:4`) in legacy CLI
```

Comment:
Does the script handle duplicate issue/PR numbers correctly (even if the change type is different)?

Reply:
Looks like it, it only groups and sorts by kind, it never uses the PR as a key. You can't have multiple news fragments for the same PR and kind together, but that's only because you can't have two `1234.feat` files at once. :)